components/kyma-environment-broker/internal/provider/azure_provider.go

```diff
@@ -19,6 +23,7 @@ func (p *AzureInput) Defaults() *gqlschema.ClusterConfigInput {
 			MachineType:    "Standard_D8_v3",
 			Region:         DefaultAzureRegion,
 			Provider:       "azure",
+			Purpose:        &DefaultAzurePurpose,
```

Comment:
do not specify the purpose here, the line is not needed

Reply:
Done!